### ISSUE TITLE
[Accessibility] Improve Alertability within Copilot Edits

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#243359]
Type: <b>Feature Request</b>

CC @meganrogge 

In Copilot Edits (in agent mode), there are some situations where users are prompted to choose the next actions.  For examples:

* Whether to continue to execute the command in terminal (this requires users' immediate gesture for the next step)

* Whether to keep the current Copilot Edits in the pending editors (does not require immediate users' attention, but this needs to be resolved before they end the current Copilot Edits session)

* Whether to finish the current Copilot Edits session (Users needd to click "Done" button)

The current issue is that these prompts are not easily noticeable  for screen reader users unless they read through the Copilot response carefully.

Is there any way to make them displayed in the Notification area such that users can hear the alert more clearly and take any required actions there?

VS Code version: Code - Insiders 1.99.0-insider (c1fb4c3983bcc28e63dd159dbbb2ea98d352d33e, 2025-03-12T08:28:07.410Z)
OS version: Windows_NT x64 10.0.26100
Modes:

**Comment by meganrogge** [comment]
@jooyoungseo , I've been wondering about that too. Perhaps when such a step is rendered, we should use aria alert to indicate it to screen reader users, which would override the response being read?

**Comment by meganrogge** [comment]
We brainstormed during the meeting:

Currently, screen reader users have hear the progress sound and don't know when the continue prompt has come in. To fix this, we need to:

- add an accessibility signal 
- indicate in the response's aria label that there is a step to take so a user doesn't have to inspect it with the virtual cursor to find this in case they missed the signal

**Comment by AFre100** [comment]
@meganrogge I'm hoping you could help me out:

• Wondering how much is this interaction related to the other related flows we've recently partnered on?
• Can you please send some footage of the interaction to kick things off?

**Comment by AFre100** [comment]
@meganrogge Apologies if I got turned around -- is a new sound still potentially needed for this?

**Comment by meganrogge** [comment]
@AFre100 , let's hold off on a sound for now. I'll discuss with @jooyoungseo and @rperez030  if that's needed - to me, the alert sounds good enough. It could just be confusing to add the sound.

**Comment by meganrogge** [comment]
To verify:

- enable a screen reader
- in agent mode, request that something gets run in the terminal
- verify you hear `Action required: run command in terminal`
- use `ctrlCmd+upArrow` to focus the response
- verify you hear `Action required: run command in terminal`
- run the action
- use `ctrlCmd+upArrow` to focus the response
- verify that you don't hear that
